login-workflow/src/pages/create-account-invite/create-account-invite.component.html

```diff
@@ -90,7 +90,7 @@
                 [activeStep]="registrationUtils.getCurrentPage()"
             >
                 <button pxb-back-button mat-stroked-button color="primary" style="width: 100px;" (click)="goBack()">
-                    {{ translate.GENERAL.BACK_BUTTON }}
+                    {{ translate.GENERAL.CANCEL_BUTTON }}
```

Comment:
Is this showing the 'Cancel' button on every page?

Reply:
This should be fixed now.